### PR TITLE
acm-4550 ki backup OADP 2.8

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
Issue: https://issues.redhat.com/browse/ACM-4550

**Review by birsanv — APPROVED** [review]
lgtm ; thanks!

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -633,6 +633,51 @@ violation - couldn't find mapping resource with kind PodSecurityPolicy, please c
 [#backup-known-issues]
 == Backup and restore known issues
 
+[#backup-oadp-failed-validation]
+=== BackupSchedule shows a FailedValidation status when using OADP 1.1.2, or later
```
Comment (oafischer) [inline release_notes/known_issues.adoc]:
```suggestion
=== _BackupSchedule_ shows a _FailedValidation_ status when using OADP 1.1.2, or later
```

**Review by oafischer — APPROVED** [review]
Looks great! One suggestion

**Pull request merged**